src/h5web/toolbar/RgbVisToolbar.tsx

```diff
@@ -4,15 +4,29 @@ import Toolbar from './Toolbar';
 import shallow from 'zustand/shallow';
 import GridToggler from './controls/GridToggler';
 import { useRgbVisConfig } from '../vis-packs/core/rgb/config';
+import Selector from './controls/Selector/Selector';
+import { ImageType } from '../vis-packs/core/rgb/models';
+import Separator from './Separator';
 
 function RgbVisToolbar() {
-  const { layout, setLayout, showGrid, toggleGrid } = useRgbVisConfig(
-    (state) => state,
-    shallow
-  );
+  const { layout, setLayout, showGrid, toggleGrid, imageType, setImageType } =
+    useRgbVisConfig((state) => state, shallow);
 
   return (
     <Toolbar>
+      <Selector
+        label="Image type"
+        value={imageType}
+        onChange={setImageType}
+        options={Object.values(ImageType)}
+        optionComponent={({ option }) => (
+          // eslint-disable-next-line react/jsx-no-useless-fragment
+          <>{option}</>
+        )}
+      />
```

Comment:
Out of curiosity, do we expect more image types? If not, a toggle group may be easier to use.

Reply:
> Out of curiosity, do we expect more image types?

Not sure but the toggle group seems indeed handier